rpcs3/Emu/System.cpp

```diff
@@ -509,6 +511,13 @@ void Emulator::Load(bool add_only)
 #endif
 
 		LOG_NOTICE(LOADER, "Used configuration:\n%s\n", g_cfg.to_string());
+		
+		// Set RTM usage
+		g_use_rtm = utils::has_rtm() && (utils::has_mpx() || g_cfg.core.enable_TSX);
+		if (g_cfg.core.enable_TSX)
```

Comment:
In order to make this universal you should use:
`if (g_cfg.core.enable_TSX && utils::has_mpx())`
in order to remove the message on supported hardware